### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by AymenFJA**
This addresses to [#3254](https://github.com/radical-cybertools/radical.pilot/issues/3254)

### CODE REVIEW

**Review by andre-merzky — APPROVED** [review]
LGTM!

File: docs/source/tutorials/submission.ipynb [hunk 1]
```diff
@@ -64,6 +64,9 @@
     "\n",
     "pd_init = {'resource'     : 'tacc.frontera',\n",
     "           'access_schema': 'ssh'\n",
```
Comment (andre-merzky) [inline docs/source/tutorials/submission.ipynb:66]: This line misses a comma, causing the NB to fail.

**Review by andre-merzky — CHANGES_REQUESTED** [review]
Left a minor change request

**Comment by mtitov** [comment]
@andre-merzky @AymenFJA have a general proposal to rewrite this section/tutorial:
1. our focus is a "local" submission - either from the login node or from the compute node (i.e., inside the batch job, using interactive job or batch script). Thus, we don't talk about `access_schema` parameters here at all, because it is set by default. We can give examples of how to get interactive job or write a batch script (while considering that the RP pilot description is not anyhow different for these examples), but we should make a note that user is required to refer to the corresponding platform's user guide to follow their instructions.
2. "remote" submission, which is **optional** and **only** if platform supports it. If it is the case, then user need to have a corresponding configuration within the resource config for that platform, and the name for that set of config parameters (e.g., `ssh`, `gsissh`, etc.) should be used in the `access_schema` for `PilotDescription`.

Thus, this section is about submission within the platform, the second part is optional

**Review by andre-merzky — APPROVED** [review]
LGTM - thanks!

**Review by mtitov — APPROVED** [review]
Let's merge it, and further updates will be in a separate PR. These changes LGTM! Thanks!

**Pull request merged**